docs/examples/getting_started.ipynb

```diff
@@ -487,6 +487,13 @@
    "source": [
     "This time, the rotation angle is randomly selected from a value range.\n",
     "\n",
+    "### Placement of argument inputs\n",
```

Comment:
I would put it also in the operators' description - this is pretty important information and having it in only one place is easy to miss.

Reply:
done